sfm_pc/models.py

```diff
@@ -76,3 +76,16 @@ def get_spreadsheet_source_field_name(cls, field_name):
             # If no source field name is specified, the default is usually
             # the spreadsheet field name with ":source" appended on
             return cls.get_spreadsheet_field_name(field_name) + ':source'
+
+
+class SuperlativeDateMixin:
```

Comment:
`get_value` seems to return an arbitrary object if there is more than one value for a complex field. This mixin adds properties to classes with start and end dates so you can retrieve the earliest start date and latest end date. This is handy for displaying information sorted by either/both dates, else the sort appears nonsensical.